src/core/src/main/java/org/apache/jmeter/gui/util/JLabeledRadioI18N.java

```diff
@@ -56,12 +56,12 @@
 
     /**
      *
-     * @param label_resouce text resource name for group label
+     * @param label_resource text resource name for group label
```

Comment:
Java convention for naming parameters is to use camel case, so I will change it.